src/pyodide/types/Pyodide.d.ts

```diff
@@ -0,0 +1,16 @@
+declare type Handler = ((...args: any[]) => any);
+
+interface PyModule {
+  [handlerName: string]: {
+    callRelaxed: (...args: any[]) => any
+  };
+}
+
+interface Pyodide {
+  _module: Module;
+  registerJsModule: (handle: string, mod: object) => void;
```

Comment:
Maybe add a TODO so we remember to make `object` stricter

Reply:
I think a js module can be any object though?

Reply:
Yeah I'm not sure what you could put here that would be more helpful.